dev-util/rgbds/rgbds-9999.ebuild

```diff
@@ -36,3 +36,12 @@ src_install() {
 	emake DESTDIR="${D}" PREFIX="${EPREFIX}"/usr Q= STRIP= install
 	dodoc README.md
 }
+
+src_test() {
```

Comment:
On merge, I'll update this to reflect phase execution order (i.e. put src_test above src_install).